src/main/java/net/neoforged/neoforge/registries/NeoForgeRegistriesSetup.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.registries;
+
+import java.util.Set;
+import net.minecraft.core.Registry;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.world.item.ItemDisplayContext;
+import net.neoforged.bus.api.IEventBus;
+import org.jetbrains.annotations.ApiStatus;
+
+@ApiStatus.Internal
+public class NeoForgeRegistriesSetup {
+    private static boolean setup = false;
+
+    /**
+     * Internal forge method. Modders do not call.
+     */
```

Comment:
The class is marked as API-internal already; this comment is redundant.

Reply:
removed